Classes/CRON/FormBuilder/Controller/FormBuilderController.php

```diff
@@ -245,7 +246,9 @@ protected function sendMail($fields, $files)
             }
 
             $customerFields = array_filter($fields , function( $v ) {
-                return $v['node']->getProperty('filter') != true;
+                if (!empty($field)) {
```

Comment:
isn't it called `$v` here instead of `$field`?

Reply:
maybe rename `$v` to `$field` in the anonymous function's parameter to be more precise and understandable?

Reply:
Yes, you are right. I have renamed the variable.